zencode/database.keys

```diff
@@ -0,0 +1,4 @@
+{
+"myUri": "sqlite:///home/apiroom/public_html/sqliteApiroom1.db",
```

Comment:
This is specific to your machine, should be corrected with a relative path(something like sqlite:///./database.db) , to make it working on all installations